src/Constraint/Bound.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+
+namespace Composer\Semver\Constraint;
+
+
+class Bound
+{
+    /**
+     * @var string
+     */
+    private $version;
+
+    /**
+     * @var bool
+     */
+    private $isInclusive;
```

Comment:
Why did you switch to this now? Thought the operator was pretty good, no?

Reply:
Bounds an always only represent `>`, `>=` or `<` and `<=`. I wanted to enforce this in the library. The array solution allowed random values and I like being explicit about things so I went for a proper class representing exactly what's possible only.

Reply:
Sure I wasn't questioning the class anyway, just whether a bool vs operator made sense, but it's fine by me :-)